src/components/web3/ExceptionListPanel.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { ReactNode, useState, Dispatch } from "react";
+import { bodyText } from "./styles";
+import { splitAddresses } from "./api";
+
+interface Props {
+  header: string;
+  subhead: string;
+  web3Address: string;
+  web3Account: "ETH" | "SOL" | null;
+  isExceptionAddressWrong?: boolean;
+  setIsExceptionAddressWrong?: (val: boolean) => void;
+  children?: ReactNode;
+  exceptionList: string[];
+  compareList: string[];
+  onChange: Dispatch<string[]>;
+}
+
+export const ExceptionListPanel: React.FC<Props> = ({
+  header,
+  subhead,
+  web3Account,
+  web3Address,
+  isExceptionAddressWrong,
+  setIsExceptionAddressWrong,
+  exceptionList,
+  compareList,
+  onChange,
+}) => {
+  const [inputText, setInputText] = useState("");
+  const [invalidAddresses, setInvalidAddresses] = useState<string[]>([]);
+  const [validAddresses, setValidAddresses] = useState<string[]>([]);
+
+  const placeholderText =
+    web3Account === "ETH"
+      ? "0x8d247168113b8aa0f020b2441e98b613b9878qmc, ..."
+      : "BtsXBPrf69qEWQrrxVENGvZY2tqFUgZB9zbrpa4t4Rfo, ...";
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputText(event.target.value);
+  };
+
+  const validateAddresses = () => {
+    const delimitersRegex = /,|\s|\t|\n/; // Regex to match comma, space, tab, or newline
+    const addresses = splitAddresses(inputText);
+    const invalidAddressesList = addresses.filter((address) => {
+      return (
+        address &&
+        (!isValidAddress(address, web3Account) ||
+          compareList.includes(address) ||
+          address === web3Address)
+      );
+    });
+
+    const validAddressesList = addresses.filter((address) => {
+      return (
+        address &&
+        isValidAddress(address, web3Account) &&
+        !compareList.includes(address) &&
+        address !== web3Address
+      );
+    });
+    if (invalidAddressesList.length !== 0) {
+      setIsExceptionAddressWrong?.(true);
+    } else {
+      setIsExceptionAddressWrong?.(false);
+    }
+    setInvalidAddresses(invalidAddressesList);
+    setValidAddresses(validAddressesList);
+    onChange(validAddressesList);
+  };
+
+  const isValidAddress = (
+    address: string,
+    accountType: "ETH" | "SOL" | null
+  ): boolean => {
+    if (accountType === "ETH") {
+      // hexadecimal Ethereum address.
+      return /^0x[a-fA-F0-9]{40}$/.test(address);
```

Comment:
Need clarity from the spec about mixed case addresses as mixed case ethereum addresses usually is to denote a SHA3 checksum.